tests/autoload.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Loads the wp-tests-config.php file.
+ *
+ * @package JWTAuth
+ */
+
+$config = getenv( 'WP_TESTS_CONFIG' );
+
+/**
+ * Supports loading the wp-tests-config.php from a non VVV custom directory.
```

Comment:
Nitpick: let's remove the VVV-specific language.

Reply:
At this point the only environment the unit tests work in is VVV, or have been tested in would be more accurate.

Reply:
@valendesigns Do you mean it can only run inside VVV? Because I was able to run it via Laravel Valet.

Reply:
![Screen Shot 2019-03-19 at 09 17 13](https://user-images.githubusercontent.com/19148962/54605347-d341de00-4a27-11e9-9f3d-0d0ffbb0115a.png)